backend/lib/endpoints/schema/auth.js

```diff
@@ -15,6 +15,11 @@ const signupSchema = {
     .prop("confirmPassword", S.string().required()),
 };
 
+const changePasswordSchema = {
+  body: strictSchema()
+    .prop("email", S.string().required())
```

Comment:
Yeah so adding the `format("email")` would be a pretty good addition
it follows these standards https://json-schema.org/understanding-json-schema/reference/string.html#email-addresses
obviously also depends on the implementation in AJV which is our validation library

Reply:
I will create another ticket for this & go ahead and merge this PR.